src/Microsoft.AspNet.Mvc.Core/Formatters/OutputFormatterContext.cs

```diff
@@ -37,5 +37,10 @@ public class OutputFormatterContext
         /// The content type which is chosen by the selected formatter.
         /// </summary>
         public MediaTypeHeaderValue SelectedContentType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the status code used by the formatter.
```

Comment:
Gets the status code that was previously set by the object result. Null indicates no value set by the object result.